framework/interface/api_handlers.py

```diff
@@ -451,6 +451,37 @@ def delete(self, target_id=None, transaction_id=None):
             cprint(e.parameter)
             raise tornado.web.HTTPError(400)
 
+class TransactionHrtHandler(custom_handlers.APIRequestHandler):
+    SUPPORTED_METHODS = ['POST']
+
+    def get(self, target_id=None, transaction_id=None):
```

Comment:
What is the need to overload get, put, patch, etc.? Don't they return 405 by default?